modules/ROOT/pages/indexes-for-vector-search.adoc

```diff
@@ -387,37 +387,78 @@ For example, you cannot have one xref:indexes-vector-similarity-euclidean[Euclid
 == Known issues
 
 The vector search index is a beta feature.
-The below table lists the known issues in its current implementation. 
+The below table lists the known issues in its current implementation.
 
-[%header,cols="5d,d"]
+[%header,cols="5a,d"]
 |===
 | Known issues | Fixed in
 
-a| xref:clauses/listing-procedures.adoc[`SHOW PROCEDURES`] does not show the vector index procedures:
+| xref:clauses/listing-procedures.adoc[`SHOW PROCEDURES`] does not show the vector index procedures:
 
 * {link-procedures-reference}#procedure_db_create_setVectorProperty[`db.create.setVectorProperty`]
 * {link-procedures-reference}#procedure_db_index_vector_createNodeIndex[`db.index.vector.createNodeIndex`]
 * {link-procedures-reference}#procedure_db_index_vector_queryNodes[`db.index.vector.queryNodes`]
+
+[NOTE]
+====
+The procedures are still usable, just not visible.
+====
 | Neo4j 5.12
 
 | Passing `null` as an argument to some of the procedure parameters can generate a confusing exception.
-a| Neo4j 5.12
+| Neo4j 5.12
+
+| The creation of the vector index skipped the check to limit the dimensionality to `2048`.
 
-| The validation for xref:indexes-vector-similarity-cosine[cosine similarity] verifies the vector's {l2-norm} can be represented finitely in {ieee-754} *double* precision, rather than _single_ precision.
+[NOTE]
+====
+Vector indexes configured with dimensionality greater than `2048` in Neo4j 5.11 should continue to work after the limitation is applied.
+====
+|
+
+d| The validation for xref:indexes-vector-similarity-cosine[cosine similarity] verifies the vector's {l2-norm} can be represented finitely in {ieee-754} *double* precision, rather than _single_ precision.
 This can lead to certain large component vectors being incorrectly indexed, and return a similarity score of `&plusmn;0.0`.
 a| // Neo4j 5.12    pending PR
 
-| The vector index `createStatement` field from xref:indexes-for-search-performance.adoc#indexes-list-indexes[`SHOW INDEXES`] does not correctly escape single quotes in index names, labels, and property keys.
+| {link-procedures-reference}#procedure_db_index_vector_queryNodes[`db.index.vector.queryNodes`] query vector validation is incorrect with a xref:indexes-vector-similarity-cosine[cosine] vector index. The {l2-norm} validation only considers the last component of the vector. If that component is `&plusmn;0.0`, an otherwise valid query vector will be thrown as invalid. This can also result in some invalid vectors being used to query, and return a similarity scores of `&plusmn;0.0`.
+
+[TIP]
+====
+For {l2-norm}alized vectors (unit vectors), thus their length image:l2norm_is_1.svg["The l2-norm of vector v equals 1"], Euclidean and cosine similarity functions produce the same similarity ordering.
```

Comment:
The "..., thus their length...," part reads awkwardly to me. Could we bracket it, or rephrase it to something like "with length ..."? I think I made a similar suggestion in the first PR, but can't remember what the outcome was. It may be that this is the proper formating, and that I have not come across it before (and need to get used to it).

Reply:
this was the compromise from before (as I copy/paste it).
Normally it would be common to have a comma between a concept and their mathematical notation `the radius of a circle, r, is the length…`, and if there are multiple listed things, you use comma and semi-colons (as a super-comma).
But that is common in academic writing … which this is not (really).

You guys had issue with the original comma and removed it.
I think perhaps we could go with:

```suggestion
For {l2-norm}alized vectors (unit vectors), thus having unit length image:l2norm_is_1.svg["The l2-norm of vector v equals 1"], Euclidean and cosine similarity functions produce the same similarity ordering.
```

I'd still want the comma, and perhaps a semi-colon; however, I know you guys don't like them :sweat_smile: